add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
```

Comment:
Observations: The form does not validate the inputs before sending them to the database.

Recommendations: Implement server-side validation to ensure that the data is in the correct format before processing it.

Example:
```php
if (empty($title) || empty($author) || !is_numeric($year)) {
    echo "Please fill all fields correctly.";
    exit;
}
```